src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,27 @@
+.forecast-summary {
+  text-align: center;
+}
+
+.forecast-summary .forecast-summary__icon i {
+  font-size: 3rem;
+  margin: 2rem 0;
+}
+
+.forecast-summary p {
```

Comment:
Related to the last comment, a good way to refactor these classes would be to use something like
`.forecast-summary__content` to target all the `p` tags, then everything that you need to overwrite with something more specific (such as `forecast-summary__temperature__content` below, which you could write as `.forecast-summary__content--temperature`)
This would enable you to target the same elements using a class you specify rather than a HTML semantic element, which is generally a better practice, plus you get to write classes using BEM methodology (which Im a big fan of and you're already halfway to using here!)

Reply:
I like this approach, thank you for sharing... I spent a while trying to research how to best structure that but the FAQs on the BEM site weren't too helpful